gnat2goto/driver/goto_utils.adb

```diff
@@ -120,6 +120,23 @@ package body GOTO_Utils is
       return Ret;
    end Symbol_Expr;
 
+   procedure New_Member_Symbol_Entry (Member_Name : String; Base_Name : String;
+                                      Enum_Type : Irep; Value_Expr : Irep;
+                                      A_Symbol_Table : in out Symbol_Table) is
+      Member_Symbol : Symbol;
+   begin
+      Member_Symbol.Name := Intern (Member_Name);
+      Member_Symbol.PrettyName := Intern (Base_Name);
```

Comment:
Why are the symbols getting interned here? In all the similar implementations of this we had the symbols where not interned.

Reply:
Copy pasted. Also wouldn't type-check, `name` is `symbol_id` not a string.